fixtures/artifactor_plugin.py

```diff
@@ -156,6 +156,7 @@ def pytest_unconfigure():
     art_client.fire_hook('teardown_merkyl', ip=appliance_ip_address)
     if not SLAVEID:
         art_client.terminate()
+        proc.wait()
```

Comment:
How long does this wait take? Maybe worth writing in the console that we wait for something if that takes longer time?